sdk/communication/Azure.Communication.CallAutomation/tests/EventCatcher/RecordedEventListener.cs

```diff
@@ -171,8 +171,9 @@ private async Task StopProcessors()
         private static string GetEventSessionFilePath(string testSessionFilePath)
         {
             // replace SessionRecords with directory name of event recordings - it'll be easier to correlate
-            var result = testSessionFilePath.Replace("SessionRecords", RecordingLocation);
-            return result;
+            var relativeEventSessionFilePath = testSessionFilePath.Replace("SessionRecords", RecordingLocation);
```

Comment:
nit: replace var?

Reply:
I decided to keep the `var` since it seems to be a pattern frequently used in this file.